src/index.js

```diff
@@ -9,7 +9,7 @@ import './index.css';
 import 'antd/dist/antd.css'; // or 'antd/dist/antd.less'
 import 'bootstrap/dist/css/bootstrap.min.css';
 import history from './history';
-import { Router } from 'react-router-dom';
+import { Router, BrowserRouter } from 'react-router-dom';
```

Comment:
Codacy found an issue: ['BrowserRouter' is defined but never used.](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-frontend/pullRequest?prid=5803328)